dev.Dockerfile

```diff
@@ -2,5 +2,5 @@ FROM node:12.20.2-buster-slim
 
 WORKDIR /app
 
-COPY package.json package-lock.json ./
+COPY package.json package-lock.json tailwind.config.js ./
```

Comment:
We actually do not need this change. The whole directory is mounted as the volume with `docker-compose-dev.yaml`.